packages/manager/src/features/Lish/Lish.tsx

```diff
@@ -110,7 +113,7 @@ const Lish = () => {
     );
   }
 
-  return linode && token ? (
+  return linode && weblish_url && glish_url && monitor_url && ws_protocols !== undefined ? (
```

Comment:
We should be able to just do 
```suggestion
  return linode && data !== undefined ? (
```

and remove lines 48-51